envs/example/swift/group_vars/all.yml

```diff
@@ -3,7 +3,7 @@ stack_env: example_swift
 swift_floating_ip: "{{ hostvars[groups['swiftnode'][0]][primary_interface]['ipv4']['address'] }}"
 
 undercloud_cidr: 
-  - 10.1.1.0/24
+  - cidr: 10.1.1.0/24
```

Comment:
@dlundquist The undercloud cidr is moving to a list of dictionaries with the keys being cidr and gateway.